gravitee-apim-console-webui/src/management/api/policy-studio/debug/components/policy-studio-debug-inspector/policy-studio-debug-inspector.component.ts

```diff
@@ -106,6 +106,9 @@ export class PolicyStudioDebugInspectorComponent implements OnChanges {
   @Input()
   executionStatus: string | undefined;
 
+  @Input()
+  condition: string | undefined;
```

Comment:
I'm not sure I understand why I have to get the condition with Input. 
it is always available in `outputDebugStep`.
maybe input & output are a bad wording name 🤔 
outputDebugStep is always the selected debug step 
and inputDebugStep is the previous debug step.

I don't think it's necessary. Or we can change RequestDebugStep to only RequestOutput to have only diff stuff

or I misunderstood something 🙈

Reply:
yes, maybe a wording problem, but it seemed more logical to put the `condition` next to the `executionContext`

Reply:
because there is also the `error` extracted from `outputDebugStep` in the same way 🤔 
admit that I don't really understand the executionStatus. this is the output status (selected debug step ^^)

I think we have to choose a solution for this 3 props. 

- always extract it (condition/error/status) and have only diff props inside input / output 
- just rename input output to avoid this non-needed extraction. But I have no idea about the name 🙄  because of App Client and Input Request ...


so 🤷‍♂️  i ✅ and as you wish or we can add a TODO and change this later